internal/cmd/kafka/command_link_create.go

```diff
@@ -266,5 +391,22 @@ func (c *linkCommand) getRemoteClusterMetadata(cmd *cobra.Command, linkMode link
 			return "", "", err
 		}
 		return remoteClusterId, bootstrapServer, nil
+	case Bidirectional:
+		bootstrapServer, err := cmd.Flags().GetString(remoteBootstrapServerFlagName)
+		if err != nil {
+			return "", "", err
+		}
+		remoteClusterId, err := cmd.Flags().GetString(remoteClusterIdFlagName)
+		if err != nil {
+			return "", "", err
+		}
+		return remoteClusterId, bootstrapServer, nil
+	default:
+		return "", "", unrecognizedLinkModeErr(linkModeMetadata.linkModeStr)
 	}
 }
+
+type linkModeMetadata struct {
+	linkMode    linkMode
+	linkModeStr string
```

Comment:
Naming is a bit redundant, i.e. `linkModeMetadata.linkModeString`
```suggestion
	mode linkMode
	name string
```